internal/provider/resource_topic_config.go

```diff
@@ -93,13 +93,11 @@ func (r *topicConfigResource) Schema(ctx context.Context, req resource.SchemaReq
 				Optional:            true,
 			},
 			"properties": schema.MapAttribute{
-				MarkdownDescription: "You can define Kafka properties for your topic here. segment.ms property needs to always be included. All options are: `segment.ms`, `retention.bytes`, `min.compaction.lag.ms`, `max.compaction.lag.ms`, `message.timestamp.difference.max.ms`, `message.timestamp.type` Read more: https://docs.axual.io/axual/2024.2/self-service/topic-management.html#configuring-a-topic-for-an-environment",
-				Required:            true,
+				MarkdownDescription: "You can define Kafka properties for your topic here. All options are: `segment.ms`, `retention.bytes`, `min.compaction.lag.ms`, `max.compaction.lag.ms`, `message.timestamp.difference.max.ms`, `message.timestamp.type` Read more: https://docs.axual.io/axual/2024.2/self-service/topic-management.html#configuring-a-topic-for-an-environment",
+				Optional:            true,
```

Comment:
Since this field is changed, documentation needs to be updated

Reply:
Fixed